sage.py

```diff
@@ -14,15 +14,14 @@
 # TODO: Find scans (code locations) whose scan frequency is higher than we recommend
 # TODO: Find signature scans taking a long time to complete (e.g. > 30m ) and suggest they be optimized, e.g. by splitting things up
 # TODO: Find signature scans that are particular large and should possibly be split up
-# TODO: Find projects that don't have any Released versions
 # TODO: Find Released versions that don't have any reports
 # TODO: Find projects that don't have an owner
```

Comment:
The change adds functionality corresponding to this line.

Reply:
i already added something to identify project-versions that are marked as Released so I figured that's enough?